src/components/atoms/button/ButtonWhite.js

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+import { ButtonType, ButtonSize } from "../../themes";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export default function Button({ size, type, label, path, onClick = () => { } }) {
```

Comment:
`ButtonType` and `ButtonSize` imports, and `size` and `type` props do not appear to be used here..

Why is `onClick` needed here if it's not used as a prop in `job-list.js`?